7.Product_Designer-FlexBox/css/style.css

```diff
@@ -0,0 +1,148 @@
+/* CSS reset */
+*{
+    margin:0;
+    padding:0;
+
+    box-sizing: border-box;
+    overflow:auto;
+}
+body{
+    background-color: #000;
+    color: #fff;
+    font-family: 'Roboto Condensed', sans-serif;
+}
+
+/* ================================TITLE CONTENT=================================== */
+.designer{
+    display: flex;
+    flex-direction: column;
+    row-gap: 130px;
+    padding: 40px 172px 0px;
+}
+/* Designer Mail */
+.designer-mail{
+    display: flex;
+    justify-content: flex-end;
+    font-size: 24px;
+}
+/* Designer Name*/
+.designer-name{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    row-gap: 6px;
+}
+.designer-name h1{
+    font-weight: 300;
+    font-size: 150px;
+}
+.bold{
+    font-weight: 900;
+}
+.designer-name h3{
+    font-size: 50px;
+    color: #cecece;
+}
+.blue{
+    color: #4990e2;
+}
+/* =================================GALLERY CONTENT================================ */
+.gallery{
+    margin-top:100px;
+    padding-right: 50px;
+    padding-left: 50px;
```

Comment:
Provide the shorthand notation of padding to reduce the number of lines
Eg.., padding: 0 50px;